app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Api::V1::CommentsController < ApplicationController
+    before_action :authenticate_user!, :except => [:show, :index]
+    
+    def index
+        @comments = Comment.all
```

Comment:
Since the comments are tied to a particular recipe, you probably don't want *all* the comments, just the ones that match `params[:recipe_id]`.